src/observers/telemetryObserver.ts

```diff
@@ -159,6 +161,17 @@ export class TelemetryObserver {
         );
     }
 
+    private handleLearnMoreClicked(event: LearnMoreClicked) {
+        let errorCode = event.code;
```

Comment:
why is this error code?

Reply:
any suggestion? that means the errorcode of diagnostic which user click the learn more button

Reply:
any suggestion? that means the errorcode of diagnostic which user click the learn more button

---
In reply to: [382360967](https://github.com/docascode/vscode-docs-build/pull/45#discussion_r382360967) [](ancestors = 382360967)